tests/botserver_mock.py

```diff
@@ -0,0 +1,170 @@
+import json
+import threading
+import time
+import websocket
+
+import handlers.handler_factory as handler_factory
+from handlers import *
+from util.loghandler import *
+from tests.slackwrapper_mock import SlackWrapperMock
+from bottypes.invalid_console_command import *
+from slackclient.server import SlackConnectionError
+
+
+class BotServerMock():
+    user_list = {}
+
+    def __init__(self):
+        log.debug("Parse config file and initialize threading...")
+        self.running = False
+        self.config = {}
+        self.bot_name = "unittest_bot"
+        self.bot_id = "unittest_botid"
+        self.bot_at = "@unittest_bot"
+        self.slack_wrapper = SlackWrapperMock("unittest_apikey")
+
+    def quit(self):
+        """Inform the application that it is quitting."""
+        self.running = False
+
+    def load_config(self):
+        """Load configuration file."""
```

Comment:
Unless this doc string adds, e.g. the format of the file, I don't think it adds anything more that we can't get from the method name.

Reply:
I don't see any value in those function docstrings either, but someone requested them earlier. From my point of view we can remove them all, but those are just from copying the existing codebase. I'll see if we can remove BotServerMock completely.

Reply:
Yeah it's nbd. It might have been me.